NEWS.txt

```diff
@@ -2,10 +2,7 @@ NEWS
 
 2.1.0
 
-- Now supporting Redis as a cache adapter
-- Local mode supports reload features
 - Configuration options for Mode and Redis parameters added.
```

Comment:
This change is not incompatible, because we have default values. should I remove it too and leave it as No news?